app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,37 @@
 <html>
   <head>
     <title>CashNest</title>
+     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+
+
     <%= javascript_importmap_tags %>
   </head>
 
   <body>
+  <nav>
+  <ul>
+    <li><%= link_to "Home", root_path %></li>
+
+  </ul>
+  <div> <% if user_signed_in? %>
+   <%= link_to "Sign Out", destroy_user_session_path, method: :delete, class: "signout-btn" %>
+    <% else %>
+      <%= link_to "Sign in", new_user_session_path, class: "signin-btn" %>
```

Comment:
Great job working on the project requirements but its been noted that you did not folllow design guidelines. IT says in the project requirements that:
Design


The student should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:


Colors
Typography: font face, size and weight.
Layout: composition and space between elements.

It would be nice to follow design guideline. Please pay attention to detail.


These are screenshots of your design:




![Screenshot 2023-08-11 at 3 26 49 PM](https://github.com/Atifzada/cash-nest/assets/79506105/3da1c305-0f09-45ad-95d7-3bdcd0fd5203)

![Screenshot 2023-08-11 at 3 25 39 PM](https://github.com/Atifzada/cash-nest/assets/79506105/5b837472-291d-43f2-a5e5-39a344c774dd)
![Screenshot 2023-08-11 at 3 26 58 PM](https://github.com/Atifzada/cash-nest/assets/79506105/443d4104-f975-4d18-a8b0-40af70894d73)
![Screenshot 2023-08-11 at 3 25 49 PM](https://github.com/Atifzada/cash-nest/assets/79506105/e1228b4a-39d9-48d4-a4f9-3a11907584fd)
![Screenshot 2023-08-11 at 3 26 10 PM](https://github.com/Atifzada/cash-nest/assets/79506105/f49a02bc-b2cf-4432-9438-aaec4deb380b)
![Screenshot 2023-08-11 at 3 26 10 PM](https://github.com/Atifzada/cash-nest/assets/79506105/464e396b-895a-407e-8532-2ecdb8e43565)

![Screenshot 2023-08-11 at 3 27 26 PM](https://github.com/Atifzada/cash-nest/assets/79506105/89424877-5cae-4d83-8162-d0f7c5cabfce)
![Screenshot 2023-08-11 at 3 26 41 PM](https://github.com/Atifzada/cash-nest/assets/79506105/4cfeb5b4-5b7c-4be6-a8a1-25c1359be028)

Please follow this design:

Your slash screen should look like this:

![Login ](https://github.com/Atifzada/cash-nest/assets/79506105/45b242ca-bd41-423b-a562-4779fa73ad99)

Your login page : 

![login2](https://github.com/Atifzada/cash-nest/assets/79506105/420e7cfe-05d8-44c8-a82d-b9d1620520f6)

SignUp  Page should look like this:

![sign up](https://github.com/Atifzada/cash-nest/assets/79506105/427b5230-c633-452c-bfc2-0ef373d33e26)

Your transaction page should look like this:
![transaction](https://github.com/Atifzada/cash-nest/assets/79506105/5978bbb7-893d-45c4-803d-9639b7f4d467)

Reply:
Sure I will work on it and improve it